news/2017-07-22-beatmap-spotlight-june-2017.md

```diff
@@ -0,0 +1,108 @@
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/illusion-of-inflict.jpg)](https://osu.ppy.sh/s/529574)
+
+This time, **[Reba](https://osu.ppy.sh/u/3087654)** is bringing us an almost perfectly structured beatmap in every aspect as well as seven guest difficulties. Although the BPM is not really high, you can find challenging patterns such as jacks, long notes, chordstreams and more, specially in **[Critical_Star](https://osu.ppy.sh/u/3793196)**'s. Talking about the hardest map in the set, it has everything mentioned before but way more dense than all the other diffculties as well as a few noticeable Slider Velocity changes at the last section of the map. Highly recommended to play since it has difficulties for every type of player, going from 1 star to almost 6 and a half.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/exit-this-earth-atomosphere.jpg)](https://osu.ppy.sh/s/575053)
+
+It's always a good surprise to find an amazing mapset as a first ranked map, and this is the case with **[Protastic101](https://osu.ppy.sh/u/6712747)**'s set! This set shows a lot of diversity: Burst, Jumpstreams, long notes... But the most amazing point of the set is the usage of Slider Velocity, empathizing a lot of slower parts and creating a really powerful feeling on the harder ones. Overall, it creates a really challenging and technical map set, sprinkled with a lot rare patterns seldom seen in the ranked section.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/rock-it.jpg)](https://osu.ppy.sh/s/569903)
+
+This map is certainly a surprise for me. Kicking off with a slowjam that slowly speeds up to your usual scroll building up the momentum with a no stop low density handstream into a technical jump-gallops with really flowy dense bursts. This map is overall a pretty nice piece hidden there in the ranked pool and I really wanted to give it the chance to be featured here. Overall just a really fun accuracy map with great technical elements and eventually some mini long notes and slider velocity changes. Fear not for those who are not capable to play this since the next difficulty from Eedow is as good an as fun to play accuracy wise but just in a lower level. Well deserved spot overall for a solid mapset.
+
+--------
+
+And that just about covers the Spotlights for this month. We hope you enjoy the new template graphics for each chart, and we hope you look forward to what we have in store next month as well!
+
+—OnosakiHito
```

Comment:
It was me but we agreed that monthly will go under his name and mine will be the themed ones (soon™).